samples/ehr-flask/katanemo/integrations/flask_auth.py

```diff
@@ -0,0 +1,69 @@
+import json
+import katanemo_sdk
+import katanemo_auth_sdk
```

Comment:
we need to name these better. its confusing a bit because both of our SDKs have something to do with auth (authentication and authorization). The second SDK is really about our dataplane (/arc/authorize) so just be named in that spirit, and the first one should capture the spirit of our control plane APIs. For example, katanemo_identity_sdk and katanemo_authorization_sdk. Please suggest better alternatives.

Reply:
naming for sdk isn't great rn. katanemo_sdk is auto-gen code from openapi, and it is primarily for control plane. I will rename it to katanemo_identity_sdk as that seems like a better name for control plane API. Will update auth sdk too.